lib/bap_traces/bap_trace_meta.ml

```diff
@@ -41,7 +51,8 @@ module Binary = struct
        @\nmd5sum: %s\
        @\n@[<2>\
        args:   %a@]\
-       @\n@[<2>envp:@\n%a@]@]@\n}" t.path (Digest.to_hex t.md5sum) Args.pp t.args Envp.pp t.envp
+       @\n@[<2>envp:@\n%a@]@]@\n}"
+      t.path (hexstring_of_bytestring t.md5sum) Args.pp t.args Envp.pp t.envp
```

Comment:
I'm wondering, why we can't use `Digest.to_hex` here? It is still in 4.06, and still has type `string -> string`.

Reply:
Ah, I understand your comment now. `Caml.Digest.to_hex` will be fine, but `Core.Digest.to_hex` will not:

```
utop[3]# #show Core.Digest.to_hex;;
val to_hex : Core.Md5.t -> string
```